.eslintignore

```diff
@@ -1,4 +1,6 @@
 /dist
 /temp
+/node_modules
 **/dist
+**/node_modules

```

Comment:
Doesnt the second ignore cover the first too?

Reply:
Yes, I just double checked it it but we have it formatted this way in every repo so I left it this way